### PR TITLE
Fixing veth iface name each time container is restarted

### PR DESCRIPTION
**Pull request by sreeman-ambati-4421**

### CODE REVIEW

File: network-wrapper [hunk 1]
```diff
@@ -292,5 +292,7 @@ if [ $DEBUG -eq 1 ] ; then
 fi
 # Remove NSPID to avoid `ip netns` catch it.
 [ -f /var/run/netns/$NSPID ] && rm -f /var/run/netns/$NSPID
+# Added a debug printout to let know that the script ran to completion successfully.
+echo network-wrapper configuration successful: $GUEST_LABEL_NAME
```
Comment (abstractor84) [inline network-wrapper:296]: The output should be json compatible, current output is json compatible, but your change is breaking that.

**Review by abstractor84 — CHANGES_REQUESTED** [review]
Need explanation for this change, not convinced with removing namespace PID from veth naming.

File: network-wrapper (hunk 1 above)
Comment (sreeman-ambati-4421) [reply network-wrapper:296]: The output is json compatible.  Have added a check based on the network-wrapper script output.

**Comment by sreeman-ambati-4421** [comment]
> Need explanation for this change, not convinced with removing namespace PID from veth naming.

The veth interfaces are not getting deleted when container is removed. Hence added this mechanism so that veth interface for a particular container is reused in next run so that stray/orphan veth interfaces are removed.

**Review by abstractor84 — APPROVED** [review]
LGTM. Hope this will not have naming conflict with mac address.

**Pull request merged**